src/components/molecules/NavigatorRowLabel/ActionsMenu.tsx

```diff
@@ -58,16 +62,19 @@ const ActionsMenu = (props: {
     );
   };
 
-  const deleteResource = () => {
+  const onClickDelete = () => {
     deleteResourceWithConfirm(resource, resourceMap, dispatch);
   };
 
   return (
     <Menu>
-      <Menu.Item onClick={cloneResource} key="clone">
+      <Menu.Item onClick={onClickRename} key="rename">
```

Comment:
hm.. should this be enabled in preview modes!?

Reply:
good catch, it should not